website/content/en/preview/getting-started-with-terraform/_index.md

```diff
@@ -18,7 +18,7 @@ Follow the clean-up instructions to reduce any charges.
 
 ## Install
 
-Karpenter is installed in clusters with a helm chart.
+Karpenter is installed in clusters via a helm chart.
```

Comment:
Why modify this?

Reply:
semantics I suppose? I can change it back